openreview/profile/webfield/archiveWebfield.js

```diff
@@ -0,0 +1,220 @@
+// ------------------------------------
+// Advanced venue homepage template
+//
+// This webfield displays the conference header (#header),
+// important instructions for this phase of the conference,
+// and a tabbed interface for viewing various types of notes.
+// ------------------------------------
+
+// Constants
+var CONFERENCE_ID = 'openreview.net/Archive';
```

Comment:
i think it should be OpenReview.net/Archive? @melisabok

Reply:
yes! please could you fix it in the live site and create a PR here? thanks